extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -71,21 +79,33 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
         this.isAmpAdElement();
   }
 
+  /**
+   * For testing purposes, this value is retrieved by function call.
+   */
+  getVisibilityState_() {
```

Comment:
If only for test, you should add @visibleForTesting however I think you could test without the need for this function.  Just create a stub that overrides /service/viewer-impl.Viewer.prototype.getVisibilityState allowing for you to control what is returns.